### DISCUSSION TITLE
Problem with Latex Generator

### DISCUSSION THREAD

**Discussion by rogta** [xournalpp/xournalpp#3584] [Q&A]
When I click on latex symbol, a box is opened and shows the following message: _failed to find latex generator program in path_.

But I already installed some software to generate pdf like fragmaster (figure 1 and 2).

Someone know how do I do to solution this issue?

Thanks community! 
![1](https://user-images.githubusercontent.com/94614044/142413356-6bdf93d2-3060-41b2-9080-186cd21661f2.png)
![2](https://user-images.githubusercontent.com/94614044/142413361-7b9f94ab-5204-464c-b1c6-926ceafaff83.png)

**Comment by rolandlo** [comment]
What OS do you use? What happens if you type `pdflatex --version` in the terminal? Have you installed a LaTeX distribution (like TeXLive, MikTeX) at all?

**Reply by rogta** [reply]
I use the linux mint. Yes, I have a LaTeX distribution (TeX Studio) installed in my pc.
![1](https://user-images.githubusercontent.com/94614044/142417501-93343a14-8636-4a80-bcc6-6d9703c08991.png)

**Comment by rolandlo** [comment]
Can you please post a screenshot of the LaTeX panel settings in menu `Edit > Preferences`?  Make sure you use the `legacy_template.tex` instead of the `default_template.tex`, because the `default_template.tex` uses the pacakge `scontents` which is only included in newer LaTeX distributions (TeX Live 2020 and later, I think).

**Reply by rogta** [reply]
Look at these...
![1](https://user-images.githubusercontent.com/94614044/142421012-b567b3ba-f060-48f1-8f72-e0aa69a331d9.png)
![2](https://user-images.githubusercontent.com/94614044/142421017-8451975e-1f86-4312-982b-96ee9243245e.png)

**Reply by rogta** [reply]
I can't find where I would change this setting you told me. Where would I change this setting?

**Comment by rolandlo** [comment]
There is a misunderstanding. Please show the LaTeX settings in Xournal++, not in TeXstudio.

**Reply by rogta** [reply]
hahaha. So sorry. I got confused.

YOU ARE SURE! I am using `default_template.tex` instead  `legacy_template.tex`, but when would I find this file (legacy_template.tex) to change in my settings?
![1](https://user-images.githubusercontent.com/94614044/142425106-0ddb4ac1-b1d4-42f9-b7d1-30309e73858f.png)

**Comment by rolandlo** [comment]
The file is in the same folder as the `default_template.tex`, hence in `/usr/share/xournalpp/resources` on Linux.

**Reply by rogta** [reply]
Wowww. I don't have this folder in my pc (figure). That Xournal folder that you can see is another version that I have in my pc. I can't find where I installed the "xournal++".

But ok, you already helped me a lot and I am sooo grateful for you spent your time with me!
   
![1](https://user-images.githubusercontent.com/94614044/142428405-2fefc48a-95bf-4f6f-afdf-83cc3fecb7c9.png)
![2](https://user-images.githubusercontent.com/94614044/142429315-864cff85-5980-4b4d-a1fb-592886832e5b.png)
![3](https://user-images.githubusercontent.com/94614044/142429319-b0015135-ac61-4036-a109-23ce075a49c1.png)

**Comment by rolandlo** [comment]
How did you install Xournal++? Do you use the flatpak maybe?

**Reply by rogta** [reply]
I had instaled by this way (fig 1).

But now I installed by this way (github) (fig. 2)

Now I got to find _legacy_template.tex_ but yet it doesn't work!

Arrrrg..!

![1](https://user-images.githubusercontent.com/94614044/142432870-51da858b-9e79-46d9-b450-b5b0499c2231.png)
![2](https://user-images.githubusercontent.com/94614044/142432874-803f88a1-1d82-4683-881d-d50517d48111.png)

**Reply by rolandlo** [reply]
Which button did you click?

**Comment by rolandlo** [comment]
Seems you installed the flatpak indeed. The flatpak is a sandboxed package, that can't use the LaTeX installation of your system. 
Either install the LaTeX extension for the flatpak via ```flatpak install flathub org.freedesktop.Sdk.Extension.texlive//21.08``` (which takes a lot of space)
or download the `.deb` package (version 1.1.0 or nightly release) from the [releases page](https://github.com/xournalpp/xournalpp/releases) and install it via `dpkg -i name.deb` where `name` stands for the `name` of the `deb`-package you downloaded.
